yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnClusterSchedulerBackend.scala

```diff
@@ -55,8 +55,8 @@ private[spark] class YarnClusterSchedulerBackend(
       val baseUrl = s"$httpScheme$httpAddress/node/containerlogs/$containerId/$user"
       logDebug(s"Base URL for logs: $baseUrl")
       driverLogs = Some(Map(
-        "stderr" -> s"$baseUrl/stderr?start=-4096",
-        "stdout" -> s"$baseUrl/stdout?start=-4096"))
+        "stdout" -> s"$baseUrl/stdout?start=-4096",
```

Comment:
executor log link order:  **stdout** then **stderr**
but, the driver log link order: **stderr** then **stdout**

After PR:

![image](https://cloud.githubusercontent.com/assets/7402327/20590908/bdb74748-b260-11e6-9a57-6f7c2f6ee61b.png)

Frankly, it is just a picky issue.